package.json

```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "13.3.4",
+    "@angular/cdk": "^13.2.0",
```

Comment:
If this version works: fine. 

Curious to know if there isn't a version `13.3.4` available

Reply:
There is 😉 (13.3.4 available)

Reply:
Updated this package and angular material to 13.3.4